components/ui/DropDown/DropDown.tsx

```diff
@@ -0,0 +1,81 @@
+import { cva, VariantProps } from "class-variance-authority";
+import { Menu, Transition } from "@headlessui/react";
+import { Fragment } from "react";
+import { ChevronDownIcon } from "@heroicons/react/20/solid";
+
+const dropDownStyles = cva("", {
+  variants: {
+    items: {
+      primary: "bg-blue-200",
+      secondary: "bg-green-200",
+    },
+  },
+  defaultVariants: {
+    items: "primary",
+  },
+});
+
+export interface MenuItemProps {
+  name: string;
+  items: string;
+}
+
+export const MenuItem = ({ name, items }: MenuItemProps) => {
+  return (
+    <Menu.Item>
+      {({ active }) => (
+        <button
+          className={`${
+            active ? dropDownStyles({ items }) : null
```

Comment:
So we should avoid the string interpolation stuff here since cva handles it for us.

Let's do something like [here](https://github.com/ninjha01/storybook/blob/headless_UI/components/ui/Toggle/Toggle.tsx#L7) where we
- [ ] create a separate function `menuItemStyles` that takes a variant `active` with a default of false
- [ ] move the string interpolation stuff into the `menuItemStyles` def
- [ ] remove the conditional logic on L32 since it's the same either way